tests/test_installer.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Science project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import subprocess
+
+from _pytest.tmpdir import TempPathFactory
+
+INSTALLER = "./install.sh"
+
+
+def test_installer_help():
+    """Validates -h|--help in the installer."""
+    assert "--help" in subprocess.check_output([INSTALLER, "-h"]).decode("utf-8")
```

Comment:
This is Python 3.12 so you can omit the encoding if you want, use `:=` match cases, etc.

Reply:
ah neat, modernized it a bit but the opportunity to use match cases eluded me - let me know if you see something I don't.